KarhooUISDK/CommonUI/Checkbox/CheckboxView.swift

```diff
@@ -16,7 +16,7 @@ class CheckboxView: UIControl {
     enum CustomConstants {
         static let visibleIconSideLenght: CGFloat = 18
         static let sideLenght: CGFloat = UIConstants.Dimension.Button.small
-        static let visibleAndActualWidthOffset: CGFloat = sideLenght - visibleIconSideLenght
+        static let visibleAndActualWidthOffset: CGFloat = 6
```

Comment:
Is it possible to use one of the pre-defined constants here? Or would it not look good?

Reply:
It's risky because the view itself is not big enough to make the difference not visible. We could go for smaller value (4), but in this case, the visible part of the checkbox will be bigger.